plugin/fzf.vim

```diff
@@ -50,7 +50,7 @@ if s:is_win
   " Use utf-8 for fzf.vim commands
   " Return array of shell commands for cmd.exe
   function! s:wrap_cmds(cmds)
-    return map(['@echo off', 'for /f "tokens=4" %%a in (''chcp'') do set origchcp=%%a', 'chcp 65001 > nul'] +
+    return map(['@echo off', 'set TERM= > nul', 'for /f "tokens=4" %%a in (''chcp'') do set origchcp=%%a', 'chcp 65001 > nul'] +
```

Comment:
fzf unsets `TERM` when `TERM=cygwin` to support msys2 (without mintty). fzf failing on mintty is intended behavior because of tcell. The exception is https://github.com/junegunn/fzf/issues/963#issuecomment-525186950 because of ssh from Unix-like OS to Windows.

What's the usecase for `TERM=xterm-256colors` on a bash shell without mintty?

Reply:
In my case, `gvim.exe` which started from mintty inherits `TERM` and it prevents running fzf.

Reply:
What if you unset `TERM` only if `has('gui_running')` is true? fzf does not work in mintty no matter what value `TERM` is set to.

Reply:
Thank you. I did it at 502432f7a6764362fa03093c0b7491140f8f3ffc.